tools/ci_build/github/android/build_aar_package.py

```diff
@@ -99,6 +101,22 @@ def _build_aar(args):
         abi_build_dir = os.path.join(intermediates_dir, abi)
         abi_build_command = [*base_build_command, "--android_abi=" + abi, "--build_dir=" + abi_build_dir]
 
+        if qnn_android_build:
+            qnn_home = os.getenv("QNN_HOME")
```

Comment:
nit: can this be read from a command line option instead? e.g., we also have a `--android_ndk_path` option. it would be easier to know how to set it if it is documented with the other options.

Reply:
@edgchen1 would you check the current way. Updated the `build_aar_and_copy_artifacts.sh` to build the command dynamically so we can add any args as needed